### DISCUSSION TITLE
ZX-VGA-JOY not working with ULA+

### DISCUSSION THREAD

**Discussion by oO-cozy-Oo** [UzixLS/zx-sizif-512#26] [General]
I`ve been using the ZX-VGA-JOY with this amazing board with no problems, but the ula+ will not show up through the interface?? The blocks that  use the ULA+ flash on screen, Its ok with the Scart lead to HDMI Converter Adapter. But the ZX-VGA-JOY is a lot sharper to me anyway so would be nicer to use this?
Cheers
Cozy
![ZX_VGA_JOY_IM3-1024x768](https://user-images.githubusercontent.com/112904233/210115216-6c4d3a4d-67ed-4208-b1eb-d31d98632bf5.png)

**Comment by UzixLS** [comment]
Does ZX-VGA-JOY support ULA+?

**Reply by oO-cozy-Oo** [reply]
> Does ZX-VGA-JOY support ULA+?
From there web site:-
This interface is not signal converter, it works in parallel with internal graphics IC(ULA) taking video data directly from CPU and provides crystal clear picture on VGA monitors.
Also, the picture on VGA monitor is precisely synchronized with PAL signal generated from ULA, so games that are relying on ULA timing looks as they should.
ZX-VGA-JOY is compatible with all Sinclair and Amstrad ZX Spectrum models and probably with a number of clones. It is not necessary to do any modification on the original computer as long as the expansion port is functional on pins used by ZX-VGA-JOY.

I`ve emailed retro-computers@elmar-el.hr ,so will post when I hear anything.

**Reply by oO-cozy-Oo** [reply]
I did get a reply from Goran who made the ZX-VGA-JOY

"ZX-VGA-JOY does not support ULA+. Being on a computer that owns a ULA+ doesn't help because the ZX-VGA-JOY creates its own image, not the conversion.
When I designed the hardware, I didn't anticipate ULA+ capabilities because I simply didn't know it existed. With small hardware changes, it wouldn't be a problem.
Although it is possible to achieve something with software, it would not be entirely good.
After doing some color palette generation tests on existing hardware, I decided that it wouldn't be entirely fair to claim that it's ULA+ even though it looks similar. So I gave up on that, at least for now."

**Reply by oO-cozy-Oo** [reply]
SOLVED!!
